scss/styles.scss

```diff
@@ -0,0 +1,95 @@
+$vars-nav-height: 90px;
+$vars-footer-height: 30vh;
+$vars-triangle-heigth: 90px;
+$vars-triangle-width: 100px;
+$vars-tech-logo-size: 100px;
+$vars-font-big: 24px;
+$vars-font-medium: 16px;
+$vars-font-small: 12px;
+
+* {
+  font-family: Roboto;
+}
+
+.front-container {
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+}
+
+.back-container {
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+}
+
+.main-title {
+  font-size: $vars-font-big;
+  position: relative;
+  margin: 100px 10px 100px 100px;
```

Comment:
we'll check this before merging, talk to me so I explain you the margin vs padding idea